packages/clerk-js/src/ui/contexts/CoreClerkContextWrapper.tsx

```diff
@@ -3,14 +3,15 @@ import React from 'react';
 
 import { CoreClerkContext } from './CoreClerkContext';
 import { CoreClientContext } from './CoreClientContext';
-import { CoreOrganizationContext } from './CoreOrganizationContext';
+import { CoreOrganizationProvider } from './CoreOrganizationContext';
 import { CoreSessionContext } from './CoreSessionContext';
 import { CoreUserContext } from './CoreUserContext';
 import { assertClerkSingletonExists } from './utils';
 
 type CoreClerkContextWrapperProps = {
   clerk: Clerk;
   children: React.ReactNode;
+  swrConfig?: any;
```

Comment:
Could you please explain why we need this?

Reply:
This is passed from `MockClerkProvider` down to  `CoreOrganizationProvider` in order to clear swr cache between tests